backend/src/email/middlewares/email-template.middleware.ts

```diff
@@ -122,9 +122,6 @@ const uploadCompleteHandler = async (req: Request, res: Response, next: NextFunc
   try {
     const { campaignId } = req.params
 
-    // switch campaign to invalid - this is for the case of uploading over an existing file
-    await CampaignService.setInvalid(+campaignId)
```

Comment:
remove in`sms-template.middleware` as well

Reply:
Thanks!